microsoft-edge/extensions-chromium/publish/create-dev-account.md

```diff
@@ -102,6 +102,9 @@ An individual account is appropriate for a developer not associated with a compa
 
 A company account is associated with an organization or business.  The account verification process is longer, and involves confirmation that you are authorized to create the account for your company.  The duration of the process can range from a few days to a few weeks.  Your company might receive phone calls from Microsoft verification partners.
 
+> [!IMPORTANT]
```

Comment:
@vchapel - fyi pls consider: https://review.docs.microsoft.com/en-us/help/contribute/markdown-reference?branch=main#alerts-note-tip-important-caution-warning - "Avoid Notes, Tips, and Important boxes. Readers tend to skip over them. It's better to put that info directly into the article text [_ie, convert to a section with a heading_]. If you need to use alerts, limit them to one or two per article. Multiple notes [_dcg team confirmed to me that they mean "alerts"_] should never be next to each other in an article."